src/store/index.ts

```diff
@@ -14,6 +19,7 @@ export interface State {
   ourSecretKey: Base64 | null;
   attestationResult: AttestationToken | null;
   uploadResult: { accessKey: string; uuid: Uint8Array } | null;
+  decryptedMessage: string | null;
```

Comment:
`executionToken` is probably a more meaningful name for this and its references? We should also probably keep it as a `Uint8Array` for now: that's what the code will actually store here, currently.

(We might later considering storing it specifically as a JWT, if we settle on that from the server side, but that's not in place yet.)

```suggestion
  executionToken: Uint8Array | null;
```